sway/commands/bind.c

```diff
@@ -27,6 +30,33 @@ void free_sway_binding(struct sway_binding *binding) {
 	free(binding);
 }
 
+struct sway_binding *sway_binding_dup(struct sway_binding *sb) {
+	struct sway_binding *new_sb = malloc(sizeof(struct sway_binding));
```

Comment:
Use `calloc` to allocate structs.

Reply:
Sure, I just copied the code over from the 0.15 branch and updated the field allocations.